src/main/java/gomedic/logic/parser/DateParser.java

```diff
@@ -0,0 +1,100 @@
+package gomedic.logic.parser;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+
+/**
+ * Class that handles formatting the date.
+ * It handles 3 different date formats:
+ * dd/MM/yyyy HH:mm (e.g. 15/09/2022 13:00)
+ * dd-MM-yyyy HH:mm (e.g. 15-09-2022 13:00)
+ * yyyy-MM-dd-HH-mm (e.g. 2022-09-15-13-00)
+ */
+public class DateParser {
+
+    /**
+     * Checks if input String is a valid time and
+     * follows one of the 2 time formats that GoMedic allows:
+     * 13:00
+     * 13-00
+     */
+    public static boolean isValidTime(String time) {
+
+        String[] timeArr;
+
+        if (time.charAt(2) == ':') {
+            timeArr = time.split(":");
+        } else if (time.charAt(2) == '-') {
+            timeArr = time.split("-");
+        } else {
+            return false;
+        }
+
+        int hour;
+        int minute;
+
+        try {
+            hour = Integer.parseInt(timeArr[0]);
+            minute = Integer.parseInt(timeArr[1]);
+        } catch (NumberFormatException e) {
+            System.out.println("Invalid time");
```

Comment:
Shall we use logger instead of printing here, you can use `logger.warn` to indicate this